src/components/NavigationList/MobileNavigation/index.tsx

```diff
@@ -1,21 +1,56 @@
 import { mobileNavigationItems } from '@data/navigation.ts';
-import Typography from '@shared/Typography';
+import SubcategoriesLinks from '@components/NavigationList/SubcategoriesLinks';
+import { ChevronDown } from 'lucide-react';
+import clsx from 'clsx';
+import { useState } from 'react';
 
 const MobileNavigation = () => {
+  const [openItems, setOpenItems] = useState<Record<string, boolean>>({});
+
+  const toggleChevron = (id: string, e: React.MouseEvent) => {
+    e.stopPropagation();
+    e.preventDefault();
+
+    setOpenItems((prev) => ({
+      ...prev,
+      [id]: !prev[id],
+    }));
+  };
+
   return (
     <div className="flex-col py-4 sm:flex lg:hidden">
       {mobileNavigationItems.map((item) => (
-        <Typography
-          tag="a"
-          href={item.url}
-          key={item.id}
-          weight="bold"
-          size="lg"
-          uppercase
-          className="flex py-4"
-        >
-          {item.name}
-        </Typography>
+        <div key={item.id}>
```

Comment:
Why do you need div here?

```suggestion
        <React.Fragment key={item.id}>
```